geode-lucene/src/main/java/org/apache/geode/cache/lucene/internal/repository/serializer/ReflectionLuceneSerializer.java

```diff
@@ -35,9 +35,9 @@
  */
 class ReflectionLuceneSerializer implements LuceneSerializer {
 
-  private Field[] fields;
+  protected Field[] fields;
 
-  private static final Logger logger = LogService.getLogger();
+  protected static final Logger logger = LogService.getLogger();
```

Comment:
Are these changes to field visibility needed? Or any of the changes to this class.

loggers should not be marked as anything but private static final. Subclasses should create their own loggers.